engine/execution/state/state_test.go

```diff
@@ -81,7 +76,7 @@ func TestExecutionStateWithTrieStorage(t *testing.T) {
 
 		sc2, update, sc2Snapshot, err := state.CommitDelta(l, executionSnapshot,
 			storehouse.NewExecutingBlockSnapshot(state.NewLedgerStorageSnapshot(l, sc1), sc1))
-		assert.NoError(t, err)
+		require.NoError(t, err)
```

Comment:
`require` can stop at the first error, which usually is the most relevant, whereas `assert` will continue running, which print lots of noise and making debugging difficult.